src/components/benefitsHub/template.test.tsx

```diff
@@ -212,4 +228,217 @@ describe('BenefitsHub with valid data', () => {
+    expect(screen.queryByText(/Online Portal/)).toBeInTheDocument()
+    const portalLink = screen.getByText('Online Portal').closest('a')
+    expect(portalLink).toHaveAttribute('href', 'https://example.com/portal')
+
+    // Check that the text-only service is rendered without a link
+    expect(screen.queryByText(/Text Only Service/)).toBeInTheDocument()
+    const textOnlyElement = screen.getByText('Text Only Service')
+    expect(textOnlyElement.tagName).not.toBe('A') // Should not be wrapped in an <a> tag
+  })
+
+  test('does not render Call us section when fieldSupportServices is empty', () => {
+    render(
+      <BenefitsHub
+        id="8"
+        type=""
+        published={true}
+        lastUpdated="2024-01-01"
+        title="Benefits Hub without Support Services"
+        titleIcon={null}
+        intro="No support services"
+        spokes={[]}
+        fieldLinks={null}
+        fieldSupportServices={[]}
+      />
+    )
+
+    // Check that the "Call us" section is NOT rendered
+    expect(screen.queryByText(/Call us/)).not.toBeInTheDocument()
+
+    // But "Message us" should still be rendered
+    expect(screen.queryByText(/Message us/)).toBeInTheDocument()
+  })
+
+  test('does not render Call us section when fieldSupportServices is undefined', () => {
+    render(
+      <BenefitsHub
+        id="9"
+        type=""
+        published={true}
+        lastUpdated="2024-01-01"
+        title="Benefits Hub without Support Services"
+        titleIcon={null}
+        intro="No support services defined"
+        spokes={[]}
+        fieldLinks={null}
+        fieldSupportServices={undefined}
+      />
+    )
+
+    // Check that the "Call us" section is NOT rendered
+    expect(screen.queryByText(/Call us/)).not.toBeInTheDocument()
+
+    // But "Message us" should still be rendered
+    expect(screen.queryByText(/Message us/)).toBeInTheDocument()
+  })
+
+  test('calls recordEvent when support service links are clicked', async () => {
+    const { recordEvent } = await import('@/lib/analytics/recordEvent')
+    const mockRecordEvent = recordEvent as jest.MockedFunction<
+      typeof recordEvent
+    >
+
+    const mockSupportServices = [
+      {
+        type: 'node--support_service',
+        id: 'clickable-service',
+        title: 'Clickable Service',
+        field_phone_number: '123-456-7890',
+        field_link: {
+          uri: 'tel:123-456-7890',
+          title: '',
+          url: 'tel:123-456-7890',
+        },
+      },
+    ]
+
+    render(
+      <BenefitsHub
+        id="10"
+        type=""
+        published={true}
+        lastUpdated="2024-01-01"
+        title="Benefits Hub Click Test"
+        titleIcon={null}
+        intro="Test click functionality"
+        spokes={[]}
+        fieldLinks={null}
+        fieldSupportServices={mockSupportServices}
+      />
+    )
+
+    // Find and click the service link
+    const serviceLink = screen.getByText('Clickable Service').closest('a')
```

Comment:
Since this is a straight-up `<a>` instead of a `<va-link>`, we can [use the preferred `.getByRole()`](https://testing-library.com/docs/queries/about#priority):
```suggestion
    const serviceLink = screen.getByRole('link', { name: 'Clickable Service' })
```